### PR TITLE
Bump plugin parent pom to 4.78

### PR DESCRIPTION
**Pull request by imonteroperez**
### Highlights
- Asking to bump JTH version to a newer version. 
- Facing issues on testing `ScriptApprovalLoadingTest.dynamicLoading` under IPv6 I faced the following issue:
```
java.io.IOException: Jenkins process terminated prematurely with exit code 1
	at org.jvnet.hudson.test.RealJenkinsRule.startJenkins(RealJenkinsRule.java:777)
	at org.jvnet.hudson.test.RealJenkinsRule.then(RealJenkinsRule.java:658)
	at org.jvnet.hudson.test.RealJenkinsRule.then(RealJenkinsRule.java:651)
	at org.jenkinsci.plugins.scriptsecurity.scripts.ScriptApprovalLoadingTest.dynamicLoading(ScriptApprovalLoadingTest.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:566)
	at org.jvnet.hudson.test.RealJenkinsRule$1.evaluate(RealJenkinsRule.java:487)
[...]
Caused by: java.io.IOException: Failed to bind to /127.0.0.1:0
```
- Missed https://github.com/jenkinsci/jenkins-test-harness/pull/712

### Testing done
- Tested locally via PCT with the proposed bump and working as expected 

```[tasklist]
### Submitter checklist
- [ ] Make sure you are opening from a **topic/feature/bugfix branch** (right side) and not your main branch!
- [ ] Ensure that the pull request title represents the desired changelog entry
- [ ] Please describe what you did
- [ ] Link to relevant issues in GitHub or Jira
- [ ] Link to relevant pull requests, esp. upstream and downstream changes
- [ ] Ensure you have provided tests - that demonstrates feature works or fixes the issue
```

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -18,7 +18,7 @@
     <jenkins.version>2.387.3</jenkins.version>
     <gitHubRepo>jenkinsci/${project.artifactId}-plugin</gitHubRepo>
     <!-- TODO until in parent -->
-    <jenkins-test-harness.version>2146.v6b_f8b_1cb_d12d</jenkins-test-harness.version>
+    <jenkins-test-harness.version>2161.v114f3cb_78326</jenkins-test-harness.version>
```
Comment (jtnord) [inline pom.xml:21]: remove workaround and update parent?
https://github.com/jenkinsci/plugin-pom/releases/tag/untagged-b1aa0dd09e60940f0c13

**Review by jglick — CHANGES_REQUESTED** [review]
https://github.com/jenkinsci/script-security-plugin/pull/559#discussion_r1489411912

**Comment by imonteroperez** [comment]
> [#559 (comment)](https://github.com/jenkinsci/script-security-plugin/pull/559#discussion_r1489411912)

Addressed

**Review by jglick — APPROVED** [review]

File: pom.xml (hunk 1 above)
Comment (jtnord) [reply pom.xml]: sorry was caugt out again by the release being there but not there.

**Review by jtnord — APPROVED** [review]

**Pull request merged**